tests/IntegrationHelper.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests;
+
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\IntegrationPartnerStatus;
+use App\Domain\Integrations\IntegrationStatus;
+use App\Domain\Integrations\IntegrationType;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+trait IntegrationHelper
```

Comment:
Maybe give this a more specific name? Something like `createIntegrationMock` like we have `CreatesMockUiTiDv1ClusterSDK`?

Reply:
Is "createIntegration" good instead of "createIntegrationMock" regarding your comment? I find including mock in the name wrong, because that is the opposite it does. It does not provide a mock, it provides a concrete implementation.